app/controllers/api/v1/visits_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Api::V1::VisitsController < ApplicationController
+  def index
+    @visits = Visit.all
+    render json: @visits
+  end
+
+  def show
+    @visit = Visit.find(params[:id])
```

Comment:
You use `find` method for take some visit by id several times

Can you please think about an option to take out this logic in separate private method

Something like that:

```
def visit
  @visit ||= Visit.find(params[:id])
end
```

And after that you can just use method `visit` in your actions instead of `@visit = Visit.new(visit_params)`